### PR TITLE
psping: add page

### PR DESCRIPTION
**Pull request by skitau**
Additional page for the useful Microsoft SysInternals tool psping. Contrib checks performed.

Happy Hacktoberfest!

### CODE REVIEW

File: pages/windows/psping.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# psping
+
+> A ping tool that includes TCP ping, latency and bandwidth measurement.
+> More information: <https://docs.microsoft.com/en-us/sysinternals/downloads/psping>.
```
Comment (owenvoke) [inline pages/windows/psping.md:4]: We've recently started removing the `en-us` locale from the doc links as the Microsoft website will redirect to the user's locale when clicking it. 👍🏻
```suggestion
> More information: <https://docs.microsoft.com/sysinternals/downloads/psping>.
```

File: pages/windows/psping.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`psping {{hostname}}:{{port}}`
+
+- Specify the number of pings and perform it quietly:
+
+`psping {{hostname}} -n {{pings}} -q`
+
+- Ping the target over TCP 50 times and produce a histogram of the results:
+
+`psping {{hostname}}:{{port}} -q -n 50 -h`
```
Comment (owenvoke) [inline pages/windows/psping.md:20]: Please could you add token syntax here?
```suggestion
`psping {{hostname}}:{{port}} -q -n {{50}} -h`
```

**Review by owenvoke — CHANGES_REQUESTED** [review]
Nice! Thanks for the page @skitau, just a couple of minor tweaks and should be good in my opinion. 👍🏻

File: pages/windows/psping.md (hunk 1 above)
Comment (skitau) [reply pages/windows/psping.md]: Ah, good call. Have addressed

File: pages/windows/psping.md (hunk 2 above)
Comment (skitau) [reply pages/windows/psping.md]: Whoops! Have updated.

**Review by schneiderl — APPROVED** [review]
Looks good thank you 👍

**Review by owenvoke — APPROVED** [review]

**Pull request merged**